roles/ara_api/tasks/main.yaml

```diff
@@ -19,6 +19,7 @@
 - name: Include OS family/distribution specific variables
   include_vars: "{{ item }}"
   with_first_found:
+    - "{{ ansible_facts['distribution'] }}_{{ ansible_facts['ansible_distribution_major_version'] }}.yaml"
```

Comment:
Maybe I was too optimistic: ``No file was found when using with_first_found. Use the 'skip: true' option to allow this task to be skipped if no files are found``